testUtils/ModelWithARandomError.js

```diff
@@ -0,0 +1,8 @@
+const Model = require('../').Model;
+
+throw new Error('some random error');
+class TestModel extends Model {
+  static get tableName() {
+    return 'test';
+  }
+}
```

Comment:
You should move this [here](https://github.com/Vincit/objection.js/tree/master/tests/unit/relations/files)